app/tests/v1/test_products.py

```diff
@@ -1,73 +1,116 @@
-import unittest 
 import json
-from app import create_app
+from app.tests.v1.base_tests import BaseTest
```

Comment:
Is there a reason for having the folder V1 inside your tests folder? And there are additional base tests files in V1 folder, what is their purpose?

Reply:
I had got an error initially and it was asking on those files, (base_testing.py), have consulted on how to deal with it. let me remove. The v1 only tests for challenge 2 in challenge 3 I will use v2 for tests

Reply:
You will need to have a new repo for challenge 3

Reply:
I didn't know that challenge 3 was to be done in another repo, I had implemented as per the guidance I was given. Challenge 3 was supposed to be in v2 as per information I got. So, let me get clear from you, I am supposed to only have app/tests/test_products.py .... and so on files, there should be no V1?

Reply:
I have resolved the unnecessary files base_test.py and base_testing.py

Reply:
cool, you can merge